packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -346,7 +345,7 @@ export const MyVideosView = () => {
       <MintNftFirstTimeModal
         shouldHideNextTime={shouldHideMintModal}
         onShouldHideNextTime={setShouldHideMintModal}
-        show={showMintModal && currentChannel?.yppStatus === 'Verified' && !mintConfirmationDismissed}
+        show={showMintModal && !!currentChannel?.yppStatus.startsWith('Verified') && !mintConfirmationDismissed}
```

Comment:
its either boolean value from `startsWith` or undefined from missing currentChannel, which is also falsy, so why would you need to use the double exclamation mark here?